app/src/main/java/org/simple/clinic/login/pin/LoginPinEffectHandler.kt

```diff
@@ -89,7 +89,8 @@ class LoginPinEffectHandler @AssistedInject constructor(
         loggedInStatus = User.LoggedInStatus.OTP_REQUESTED,
         registrationFacilityUuid = entry.registrationFacilityUuid!!,
         currentFacilityUuid = entry.registrationFacilityUuid,
-        teleconsultPhoneNumber = entry.teleconsultPhoneNumber
+        teleconsultPhoneNumber = entry.teleconsultPhoneNumber,
+        capabilities = null
```

Comment:
Setting `null` here is wrong, we need to update `OngoingLoginEntry` to have capabilities as well. Please refer `UsersApi#activate` API endpoint which will return `LoggedInUserPayload` which will have the capabilities.